server/utils/ssr.tsx

```diff
@@ -68,6 +68,7 @@ export const generateMetaComponents = (metaProps: MetaProps) => {
 	} = initialData.communityData;
 
 	const url = `https://${initialData.locationData.hostname}${initialData.locationData.path}`;
+	const isPub = !!initialData.scopeData.elements.activePub;
```

Comment:
the check on scopeData is needed because, I believe, the test data in ssr.test.tsx is out of date and does not include scopeData. We could fall back to checking the url for the existence of `/pub/` for determining if it's a pub if that feels better.